packages/main/src/Calendar.ts

```diff
@@ -182,8 +182,10 @@ type SpecialCalendarDateT = {
  * @csspart day-cell-selected-between - Used to style the day cells in between of selected dates in range.
  * @csspart month-cell - Used to style the month cells.
  * @csspart month-cell-selected - Used to style the month cells when selected.
+ * @csspart month-cell-selected-between - Used to style the day cells in between of selected months in range.
```

Comment:
we have to export the parts in the hbs of the `ui5-daterange-picker`

Reply:
resolving this because the conversation is continued in the comment below